src/lib/markdown-converter.ts

```diff
@@ -14,5 +14,8 @@ export function convertToSanitizedHtml(markdownContent) {
     return '';
   }
   const unsanitizedHtml = showDownConverter.makeHtml(markdownContent);
-  return sanitizeHtml(unsanitizedHtml, sanitizerConfig);
+
+  const unsanitizedHtmlWithRenamedLinkForOpenInNewTab = unsanitizedHtml.replace('<a', "<a target='_blank' ");
```

Comment:
kanskje endre markdownContent før konvertering til html, da får du mindre forvirrende variablenavn

Reply:
Mener du at vi skal parse markdown som denne:
[I'm an inline-style link](https://www.google.com)

For å putte inn en inline-html som denne:
<a target='_blank' href="https://www.google.com">I'm an inline-style link</a>

Før vi kjører markdown?

Reply:
Mener du at vi skal parse markdown som denne:

`[I'm an inline-style link](https://www.google.com)`

For å putte inn en inline-html som denne isteden:
`<a target='_blank' href="https://www.google.com">I'm an inline-style link</a>`

Før vi kjører markdown?

Har testet dobbelt parsing og dette fungerer:
`<a target='_blank' target='_blank'  vhref="https://www.google.com">`